src/category_theory/monad/kleisli.lean

```diff
@@ -93,4 +95,61 @@ nat_iso.of_components (λ X, iso.refl _) (λ X Y f, by { dsimp, simp })
 end adjunction
 end kleisli
 
+@[nolint unused_arguments]
+def cokleisli (U : comonad C) := C
```

Comment:
Deserves a docstring! 

The docstring on `kleisli` is now out of date - it should say 
"Kleisli category of the monad `T : monad C`" - the definition now is only for monads; feel free to make that change in this PR if you like